prevodach.js

```diff
@@ -0,0 +1,48 @@
+const axios = require('axios').default;
+const { v4: uuidv4 } = require('uuid');
+
+// PLEASE DONT STEAL THIS
+// GCP SECRET VALUE ?
+var subscriptionKey = process.env['translator-key'];
+var endpoint = 'https://api.cognitive.microsofttranslator.com';
+
+export async function swedishToEnglish(message) {
```

Comment:
`export` is not supported in node. If we want to use `export` & `import` instead of `exports.module` & `require` we need to add a build step, transpiling from es6 -> js.

Replace this line with with....


```suggestion
module.exports.swedishToEnglish = async (message) => {
```

And I think you'll get the functionality you want

Reply:
tfw always writting ts dont even know export is not supported in vanilla node